core/client/app/templates/settings/labs.hbs

```diff
@@ -47,11 +47,10 @@
             <fieldset>
                 <div class="form-group for-checkbox">
                     <h3>Enable Beta Features</h3>
-                    <label class="checkbox" for="labs-publicAPI">
-                        {{input id="labs-publicAPI" name="labs[publicAPI]" type="checkbox" checked=usePublicAPI}}
-                        <span class="input-toggle-component"></span>
-                        <p>Public API - For full instructions, read the <a href="http://support.ghost.org/public-api-beta/">developer guide</a>.</p>
-                    </label>
+                    {{gh-feature-flag
+                        flag="publicAPI"
+                        helpText='Public API - For full instructions, read the <a href="http://support.ghost.org/public-api-beta/">developer guide</a>.'
```

Comment:
I think it would make more sense to pass the `gh-feature-flag` component a block here so that the help text can be supplied a bit more naturally? If we do that and pass a positional param instead of a named param the invocation becomes:

```
{{#gh-feature-flag "publicAPI"}}
    Public API - For full instructions, read the <a href="http://support.ghost.org/public-api-beta/">developer guide
{{/gh-feature-flag}}
```